data/assets/scene/digitaluniverse/exoplanets.asset

```diff
@@ -32,21 +32,15 @@ local Object = {
     },
     Unit = "pc",
     SizeSettings = {
-      ScaleExponent = 16.9,
-      MaxSize = 2.8,
+      ScaleExponent = 16.4,
```

Comment:
I suggest increasing this value again.  This makes the rings very small on my display